### ISSUE TITLE
Crash on large fetch

### ISSUE THREAD

**Issue by nyurik** [sqren/backport#65]
Recently Kibana had a huge pull request merge. Running backport afterwards resulted in this stack trace.  Subsequent running kinda got it back in order.

```
$ ./node_modules/.bin/backport 
? Select commit to backport Bumped Vega lib version to 3.3.1 (#17829)
? Select branch to backport to 6.x
Backporting #17829 to 6.x
✖ Pulling latest changes
{ Error: stdout maxBuffer exceeded
    at Socket.<anonymous> (child_process.js:252:14)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:547:20)
  cmd: 'git reset --hard && git clean -d --force && git checkout master && git pull origin master' }
```

**Comment by sqren** [comment]
Thanks for the heads up. `exec` could be replaced with `spawn` but bumping `maxBuffer` should fix it for now.

**Comment by sqren** [comment]
Published as 3.0.2
